scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -245,14 +245,24 @@ fi
 
 # Let's build all the things!
 
-temp_dir=$(mktemp --directory)
+# multipass become a strict-confined snap after 1.3.
+# this check is to make sure the temporary folder will be using
+# the allowed locations when copying files to multipass 
```

Comment:
is there any reason to not just always use this logic with multipass? It's not like the multipass snap will ever go back in time to being a classic snap, so this logic will really just be for old installs stuck on the classic version of the multipass snap, and the same logic works for classic and strict snaps, so I think you could actually simplify this branch to just always use --tmpdir="$HOME".

Reply:
Gotcha. I will try to solve the issue with just $HOME